tutorials/mantisshrimp-fasterrcnn.ipynb

```diff
@@ -0,0 +1 @@
MLFy5JgGeS0zSOdbliuViwXMzJsgStFX/yJ3/CD77/PbwrUZJAQwgdg1HOG197k298+9u8+fbb1M6RDRO29sYYo1muZ3ROSO4OH4mu4ucj3aCzFHai5Li3i/c+RGdVBT3vIQjDQXkDXtKonY/Gpv0t7ZPeg0KiPQyT0Q5c8Dx6eI9/8o//mPFoyNfevsOtm6/QtQ5lTpkv11St5CxVdcfs9ITZ8xl11YkTdZB4DRWRHUK3aV0o1W42SiHruthONBDO1EnG6Kh6UoTQ0DUOl0i+k3BEEsRI0OI8ZMUOZbnk7meHnB7/GacnS77+zpvcuX0LDDx49AgXxJK+6p7S1S3BBWbzKT/84ffJbMJbb72N1qk4aBvL/t4BN19+hedHxzy4f19aawE6X8vKcW6j6bpuowD0TY/C2HOnaDkAeBULufOblpaWRV1XaCscP2uFw9O10tK7cf0qr796hz/5oz/m8NlzjA4boqcQ+8/xheIftJb24eYnZ2gbcfWz5qxQ6DOvzvseyTpkSGJLVlCXMw5OrDdFnReJ8AQht1dlRblcUTclly9fZHd3j+FoyGKx4jvf+e6mwGqaBuc7jE64ER0Bbt26zWKxoK5rlAqkSSLtlbhpB2RfingEBE3owNiELM8IoxZXV9S1IzhRUDqnME6jQsZwWLA1HssGHL2piKjlBkPq51Z8ThLTIIV/VTqCNxiT0TWexXzNBx98xPHxKW8cHvKVr36Fb7z7LrtbE374gx9w/8E96lbQ/VSkQy/szy+29aR1reMJxXu3SW4WtMVgVSqIXPQog4xhkZIVFucaJJlds7t/idVoxXy2IrhAnp21b54dHsnnci1t1zKbLxienjIcjhgNhTd2crzg9PSUqlygVSBJE8q6wiMhsWXpopihw7X+fMoK4/GY49MpP+/6pUXLaJTRtpClI7IsIx8UHD474vDwCNd2tK3DhZrOeZrGMRgUZLmQktrW0TkfpX9ikuSdx3WS89A2Da1rRCffcdHGdwAAIABJREFUM6Q3Ue8KRyDLMnZ2dxkUeSRIpmBOKdspLroB2iRB6xSTpKRFLnksWuFcS54mWK3JrKJILKlSJEoxGBQEBbPFguezU3rBVVIIkVJFlULYEF/jidX00zI6WgZZBBSKNM1IkoTVasXz4yn37t9nvV6j85TZbAahw5kaZba5eGmfpilpO0eSiDdD12pWq5LFYoWmItEJmU1JMkuLRylD0H6zUcoE9JuBa1R8z95vTlNd58jzgjRNadpWUKxzRNS2bamqKpKl60j6Eh+Jw+fPqH5Y43zgzqtf4eKly6RWotKVE8Ts0oUrDAdDfvR+xuMn9yUM0MYCQYnHjHPiOaFdvwEJ8pKmhmFiSUzGZGvI/v4lfGd5cP8ROoDVGu0Vq9kKHUogwTvHwcEl9ncvsj3e4bt/9R3u3/+MqiojSVMWwuB8tIqXLBTVZ9wEMQNMkoTt7R0uXbrAcDhktlhyfHyMdzVN2zslG7qAJDGvS5SyUYIqnwdCPIHEFpiSzd0YxXI155OffEQIgW9961d5+eWbsrh5T1PVKMQ9MkmE0f/GV7/Co8cPefjwM/G/MZ4k09jMcDI75uHj+4z3JqSDAetygQ8triN6xIgkORA3IOnAbTaJfnxIQeWENK6FGGuUjfEVkrOSWB0LmbOUWdUhuTbySemco2lb8ZbxQQwbO7nPWTpgPjvlH/39P6Rtlrz1zptcv34Nj4++FmJOqVvPoMio05Sm7GF7EzmiEkGhjInp23GDI8TdTj6j9wGcmBxGNrHwWkRHBDYeKEI/TwxKGcnRSiRyQGkjAYpt4P6DZzx9+gd8+eUDvvneW1y7cllMsBQEZP6cHB1Tr2u6ruXo+SEffPg+RVFw6dJVtM3wypMUGVcvX2H6ym2ePnrKbHoq7123hLg6989F9pdIhuiJyn2NgHBiFB5xIY/pSx5Qkk7sfcu6nFO3K1CQYKDt8HQo4wiqYzBMmUwGPHvWSpsjuDOr+CDFnhQC8tVqLc64qvfYOUfaDR5jUnoEtV8Ne4k68XsffEQxDNZmNI0WQ8v4JGODSH53fJUQRCxBEE+gZ0+fcTo7ZX9vX/LbFnOms1OGwyHjySS2TV7k17z66mvcu3eP5XJJkvRqGsl5sla4Ud6fvV9pemrhRxnNcDgkuB1OTjrqai3jxgWUMWirKAY5o/GIoCS6gb5d03MLN7V/RFhCiF4lgabqKNcSMYPXgoQivMbDZ4fM5zOePnrMN775TW7fusVwMMAkhs+/uLuxJgAh4fZKoZ+2++8tBPohJYGVEntQ5IbE5gyHgdVyTtPUJLXCakeaqo1jtlKKPM9ITQpYdrd2+PrbZ54v05MZk60xg2LCxQMl7vcmh2ApS49zOqooK5pmKYT6QuIs2q6lc5Ec72C9LDdrWH8NBwPMX89o3Fy/tGi5eu0S63VLmgw4UCnOaZ48PiR88BHPj0+o6xrnPWVZ03UdZVlK/01Jfk7X+SiJlbwK33V4H6R3HcMVexQDNB5F50M8JRrQKSYboJIMrxWN1zQu0HSexkm+ibVE+VcSA9c8eWrxDoosochTBtaSJQkGqbTLrmV6csKqbmjPeVKLpFJOkiFIEJWsI5EHoBANfx/prTRGidtvlmUEH3g+fc7nn9/n8PBQIrl9S7MWkpS3HZOtIVmW0rYpnWuwVpNYy3pZsVgsqeuOVAXKxZp1umJrT2ziG9+ibILanHLUpnAh9FyEM+xVig9HlljyPJcJ7NzmRC0D3cdqvMV7HU/RfoNKTE+P+fH735fMIeDy1SuyOZkU72raqmF3e5+333yXxCZ89vknONeRZpY8TwnB07YN0r4K8QRydlosBgPGw5yLlw5444230OQEfsB0NqVuIE9Tbl5/jWsvvUrTpZzOHShLMZ7w67/+67x0/Tp//ud/yocffsjx8yPaZo3R0iIQxYMUFH0l730gcMYjcM4xHk+wWcFiuWa2WMmpXfUbiKJpOkIoARVNB7uo9DjLgHJeXF
```

Comment:
<p>It would be good to used code formatting like <code>FasterRCNNParser</code>:  better readibility</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lgvaz/mantisshrimp/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Yes, reformmatted</p>